app/controllers/hub/clients_controller.rb

```diff
@@ -97,7 +97,7 @@ def update_take_action
     end
 
     def unlock
-      raise CanCan::AccessDenied unless current_user.admin?
```

Comment:
I could be convinced to break this into two lines or a way to make it read easier.